packages/waterfall/package.json

```diff
@@ -0,0 +1,7 @@
+{
+  "name": "@mui/internal-waterfall",
+  "version": "1.0.0",
```

Comment:
Usually we use "5.0.0" for the private packages, but we can do the change once for all packages in a follow up PR.